<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Delete_score</class>
 <widget class="QMainWindow" name="Delete_score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#Delete_score
{
	
	border-image: url(:/pic.png);
	background-color: rgb(255, 255, 255);
	
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="delete_student_id">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 230, 255);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 12pt &quot;宋体&quot;;
color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Places input student id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete_return">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QMainWindow#Delete_score QPushButton#btn_delete_return
{
   font: 18pt &quot;宋体&quot;;
	
	background-color: rgb(255, 255, 255);
color: rgb(0,0,0);
}

QMainWindow#Delete_score QPushButton#btn_delete_return:hover
{
	
	background-color: rgb(195, 195, 195);
}</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete_confirm">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QMainWindow#Delete_score QPushButton#btn_delete_confirm
{
	
	background-color: rgb(174, 255, 186);
  color: rgb(0,0,0);

border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 18pt &quot;Candara&quot;;

}

QMainWindow#Delete_score QPushButton#btn_delete_confirm:hover
{
	background-color: rgb(120,200,150);
}</string>
    </property>
    <property name="text">
     <string>确认删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="delete_title">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>删除学生成绩界面</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
